### ISSUE TITLE
Support AWS Certificate Manager as a new provider

### ISSUE THREAD

**Issue by AndreyZamyslov** [external-secrets/external-secrets#839]
We would like to use External Secrets to deliver certificates directly from [AWS Certificate Manager](https://aws.amazon.com/certificate-manager). But it looks like AWS Certificate Manager is not supported as a provider now.

Can it be supported? Or maybe there is some reason why it shouldn't be?

**Comment by AndreyZamyslov** [comment]
Of course our case is not representative but we are going to deliver only certificates signed by a private CA

**Comment by moolen** [comment]
hmm, i'm just wonder would the cert-manager integration work for your use-case: https://github.com/cert-manager/aws-privateca-issuer ?

**Comment by AndreyZamyslov** [comment]
Thanks for suggestion! We're considering various options.
The problem is that we've used 2 cloud providers: AWS and Yandex.Cloud. Probably some new ones will be added in the future. And we would like to use a similar approach to deliver secrets and certificates from all the clouds.

We've already delivered secrets with External Secrets and like it. I plan to prepare a PR to add support for Yandex.Cloud Certificate Manager within a couple of weeks and it seems great to have an ability to take certificates from AWS Certificate Manager too.

Anyway it's not a final decision and not very critical at the moment. We keep thinking about how to be with AWS. If we decide it's important I think we could create a PR to support AWS Certificate Manager (if that is welcome)
